### DISCUSSION TITLE
howto: png logo with transparancy over another image

### DISCUSSION THREAD

**Discussion by ddnddn** [ImageMagick/ImageMagick#2041] [Help]
hello
I cannot get this to work as simple as it is.

I have to images, a png logo without background that i want to lay on top on another. No matter what I do the logo become rectangled, can someone help, so much thank you.

kind regards,
Danny

**Comment by fmw42** [comment]
Please post your input image and if possible show your desired output and what you are getting. 

And your exact command line that you are trying to use.

Please always provide your IM version and platform, since syntax differs.
